webargs/core.py

```diff
@@ -245,9 +248,13 @@ def _validate(self, name, value):
                     val = ret[key]
                 except KeyError:
                     if nested_arg.required:
-                        raise RequiredArgMissingError(
-                            'Required parameter "{0}" not found.'.format(key)
-                        )
+                        req_type = (nested_arg.required).__class__.__name__
+                        if req_type in ['str', 'unicode']:
```

Comment:
Yeah, I wasn't sure what's the best way to go about this. I've seen other people using the `six` package for this purpose, which obviously is not a solution...

I'll give it a try using `basestring`.